PackageViewModel/PackageAnalyzer/NativeTargetsFilenameRule.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.IO;
+using System.Linq;
+using NuGetPackageExplorer.Types;
+using NuGetPe;
+
+namespace PackageExplorerViewModel.Rules
+{
+    [Export(typeof(IPackageRule))]
+    internal class NativeTargetsFilenameRule : IPackageRule
+    {
+        #region IPackageRule Members
+
+        public IEnumerable<PackageIssue> Validate(IPackage package, string packagePath)
+        {
+            var targetsFiles =
+                package.GetFiles().Where(x => x.Path.EndsWith(".targets", StringComparison.OrdinalIgnoreCase));
```

Comment:
Should this be `.props` as well?

Reply:
The rule now validates both .targets and .props.